docs/src/app/(public)/(content)/react/components/avatar/demos/hero/css-modules/index.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Avatar } from '@base-ui-components/react/avatar';
+import styles from './index.module.css';
+
+export default function ExampleAvatar() {
+  return (
+    <Avatar.Root className={styles.Root}>
```

Comment:
We can extend the hero demo so also show an Avatar with initials only, something like:

```
<Avatar.Root>
  MN
</Avatar.Root>
```

Reply:
Also, I would make the avatars a bit smaller :)